src/test/java/com/gepardec/mega/service/impl/monthlyreport/MonthlyReportServiceImplTest.java

```diff
@@ -43,34 +59,58 @@ class MonthlyReportServiceImplTest {
     @Inject
     MonthlyReportServiceImpl monthlyReportService;
 
+    @InjectMock
+    private EmployeeService employeeService;
+
+    @InjectMock
+    private UserContext userContext;
+
+    private MockedStatic<UserContext> mockStatic;
+
```

Comment:
Make all fields package-private, since IDE is confused about them not being instantiated explicitly